src/logic.js

```diff
@@ -487,8 +489,26 @@ define([
                     load[path] = [prop];
                 }
             });
-
-            return {load: load};
+            var saveToCaseQuestions = _.filter(this.form.getMugList(), function(mug) {
+                return mug.__className === "SaveToCase";
```

Comment:
I'll PR the second half of this now, but main use case was to get this info in app summary, exports, and data dictionary which is a huge win for enterprise and GS efficiency. I had actually originally thought i'd do it in HQ and @emord convinced me to do it here. [some discussion here](https://docs.google.com/document/d/1VE0vtYNiR0xkyXjos9YfVgH6f6Dii4kxLnhnqT3ELuI/edit#)